Sources/Remote Logging/EventLogging+Sentry.swift

```diff
@@ -22,13 +22,18 @@ extension EventLogging {
             return
         }
 
+        guard let timestamp = event.timestamp else {
```

Comment:
The `timestamp` property was `Date` in 6.x but is `Date?` in 7.x

Reply:
The previous implementation (see diff below) used a `Date()` fallback here.

I wonder which is more appropriate:

- Handle the case of a missing timestamp (how likely is that?) by using an incorrect one, hence cancelling the timestamp value as a way to determine the correct log file; or
- Bail out because an event without timestamp is inconsistent / unexpected

🤔 

Keen to hear your thoughts. The fix to support falling back to `Date()` is straightforward for me:

```swift
let timestamp: Date
if even.timestamp == nil {
    let now = Date()
    TracksLogDebug("📜 Unable to locate event timestamp. Falling back to current date (\(now))")
    timestamp = now
} else {
    timestamp = timestamp
}
```